python/thrift/appengine-ssl-gateway/README.md

```diff
@@ -0,0 +1,253 @@
+# Connecting from Google App Engine to Google Cloud Bigtable
+
+It's easy to deploy a Thrift gateway cluster on Google Compute Engine that
+proxies request to the Cloud Bigtable API.
+
+Here are step by step instructions for starting a Cloud Bigtable cluster and
+building and deploying a Thrift server on Compute Engine.
+
+**Table of contents**
```

Comment:
The alternative to using `make` is copy-pasting a bunch of longer command lines, with no error checking: e.g., if an environment variable is unset, you get a really inscrutable error which is hard to debug, as it doesn't actually say that the error is that the variable is being defaulted to the empty string.

If there's a better way of avoiding the rough edges for users, I'd love to use it, but this is the best I came up with to simplify the user experience and catch any errors (typos, forgetting to set env vars, etc.).